programs/mpl-core/src/utils.rs

```diff
@@ -535,23 +550,23 @@ pub(crate) fn validate_collection_permissions<'a>(
         }
     };
 
-    if let Some(external_plugin_validate_fp) = external_plugin_validate_fp {
-        match validate_external_plugin_checks(
+    if let Some(external_plugin_adapter_validate_fp) = external_plugin_adapter_validate_fp {
+        match validate_external_plugin_adapter_checks(
```

Comment:
One could argue that at the time lifecycle validation is actually occurring, it is now in fact a full external plugin because the authority has fully created the plugin.  And thus at this point it is not an adapter only, and then it would still be more appropriate to say `validate_external_plugin_checks` here